test/internet/test-http-dns-fail.js

```diff
@@ -4,7 +4,6 @@
  * should trigger the error event after each attempt.
  */
 
-var common = require('../common');
```

Comment:
That looks a bit strange imho and would confuse someone who doesn't know about this 'dependency'.